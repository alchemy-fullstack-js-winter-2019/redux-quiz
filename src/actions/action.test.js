import { UPDATE_TITLE, updateTitle } from './action';

describe('action test', () => {
  it('updates title', () => {
    const action = updateTitle('Real Title');

    expect(action).toEqual({ 
      type: UPDATE_TITLE,
      payload: 'Real Title'
    });
    expect(action).toMatchSnapshot();
  });
});