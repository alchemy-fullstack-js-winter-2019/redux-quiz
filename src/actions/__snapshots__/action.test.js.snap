// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`action test updates title 1`] = `
Object {
  "payload": "Real Title",
  "type": "UPDATE_TITLE",
}
`;
